managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/PositionImpl.java

```diff
@@ -94,6 +94,14 @@ public PositionImpl getNext() {
         }
     }
 
+    public PositionImpl getPositionAfterEntries(int entryNum) {
+        if (entryId < 0) {
```

Comment:
This is not correct

Reply:
Sorry, didn't understand.
delete this judgment?

Reply:
Fixed, PTAL,thanks! @eolivelli

Reply:
add check :
    public PositionImpl getPositionAfterEntries(int entryNum) {
        if (entryNum < 1) {
            return this;
        }
        if (entryId < 0) {
            return PositionImpl.get(ledgerId, entryNum);
        } else {
            return PositionImpl.get(ledgerId, entryId + entryNum);
        }
    }